test/index-test.js

```diff
@@ -5,12 +5,12 @@ const got = require('got');
 describe('Server', () => {
     const myServer = new app.MyServer();
 
-    before(() => {
-        myServer.listen();
+    before(async () => {
+        return await myServer.listen();
```

Comment:
Well, it's fine as is, but you don't have to use `return` here. Since `async` functions always return a Promise implicitly.